kubectl-cloudflow/cmd/deploy_command.go

```diff
@@ -373,3 +376,42 @@ func validateDeployCmdArgs(cmd *cobra.Command, args []string) error {
 
 	return nil
 }
+
+// The function validates that the operators for Spark and Flink is installed if the application uses any of those streamlet types
+func validateStreamletRunnersDependencies(applicatonSpec cfapp.CloudflowApplicationSpec) {
+
+	const requiredSparkVersion = "v1beta2"
+	const requiredFlinkVersion = "v1beta1"
```

Comment:
maybe these consts can go in a more central place, we have a version.go somewhere I think.

Reply:
Moved to version